src/main.rs

```diff
@@ -1,10 +1,13 @@
 use rall::{debug, error, fatal, info, trace, warn};
 
 fn main() {
-    trace!("My Best Friend Hazel :D");
-    debug!("My Best Friend Hazel :D");
-    info!("My Best Friend Hazel :D");
-    warn!("My Best Friend Hazel :D");
-    error!("My Best Friend Hazel :D");
-    fatal!("My Best Friend Hazel :D");
+    trace!("My Best Friend Hazel {}", ":D");
+    debug!("My Best Friend Hazel {}", ":D");
+    info!("My Best Friend Hazel {}{}", ":", ")");
+    warn!("My Best Friend Hazel {}", ":)");
+    error!("My Best Friend Hazel {}", ":P");
+    fatal!("My Best Friend Hazel {}", ":D");
+
+    let very_important_value = String::from(";)");
+    debug!("Look at this: {}", very_important_value);
```

Comment:
I would just make it so that the main function looks like as it was before but with the formatting.
E.g.
```rust
trace!("My Best Friend Hazel {}", ":D");
debug!("My Best Friend Hazel {}", ":D");
info!("My Best Friend Hazel {}", ":D");
```